### ISSUE TITLE
Export to excel errors

### ISSUE THREAD

**Issue by JohnReinhart** [DrCoffey/DeepSqueak#164]
I am using DeepSqueak 3.0.1 on MatLab 2021b. I have processed 14 files and exported them to excel without error but for one litter of 3, recorded at the same session as another litter that has processed successfully, I receive errors when trying to export to excel. I have attached a pdf with screenshots of the errors I receive and was curious if you had any suggestions? Each file is hand scored where I identify and classify each call myself from WAV files from Avisoft.
[DeepSqueak Errors.pdf](https://github.com/DrCoffey/DeepSqueak/files/8326023/DeepSqueak.Errors.pdf)

**Comment by JohnReinhart** [comment]
https://docs.google.com/presentation/d/1Jg1luBJf9HoF7i6TPMxp0ztRYOpNjkDSR4T3592G9JM/edit?usp=sharing

**Comment by DrCoffey** [comment]
That error usually only occurs when a box is screwed up and outside the Audio range, but because it is happening for every call there must be something else going on. Are you using separate folders for audio files and detection files? Have you set both of those folders before trying to export the stats?

**Comment by JohnReinhart** [comment]
Yeah, each animal in the litter has its own folder processed individually. Each file produced by DeepSqueak, before being converted to an excel file, is saved within the same folder that represents each animal.

**Comment by DrCoffey** [comment]
Got it. Usually I would keep all of the audio files for 1 project in a single folder, and all of the detection files for that project in a separate folder. That way you can analyze the files in a batch. You could then export them all at once or cluster them all at once. 

Either way the audio file and detection files need separate folders, and you need to point DeepSqueak to those folders before exporting.

**Comment by JohnReinhart** [comment]
I did try saving the detection files in one separate folder at one point but was still getting the same error.

**Comment by DrCoffey** [comment]
It is likely something strange with the one detection file then, without seeing that specific file and corresponding audio file, it will be hard to tell what is going wrong. If you can share them I might be able to help.

**Comment by JohnReinhart** [comment]
It wont allow me to load zipped files, any good suggestions on how to share them with you?

**Comment by DrCoffey** [comment]
Google drive links usually work well.

**Comment by JohnReinhart** [comment]
https://drive.google.com/drive/folders/1NbVNSLRftOq076huDaNPvNxbZoNsdkSP?usp=sharing

**Comment by JohnReinhart** [comment]
We found a workaround in the excel_Callback function. In effect what was happening was that one of the "if" statements which checked the characteristics of the calls (start time, end time, and length of call) was triggering when the data was being migrated out of Deepsqueak and into the excel sheet output. The reason for this was unclear as the call duration was within acceptable limits. By turning off this part of the warning we were able to get the excel sheet to populate correctly with values that make sense. We tried to replicate it with different files, but there wasn't any apparent similarities between the several files which failed to export. Attached is a screen capture of the change that was made to stop the error message from appearing and blocking the output of the excel sheet. 
![Screenshot 2022-04-07 153434](https://user-images.githubusercontent.com/101817767/162291349-7d800947-2cb8-466d-a5ac-aa277b620c4b.png)
